1908-01-30.xml

```diff
@@ -1923,28 +1923,5053 @@
+                            l'Egypta," and</p>
+                        <p>2. Because the opening of busine.in Cyprus was voted in due format a
+                            general meeting of shareholders.</p>
+                        <p>With regard to the first contention maintain that Cyprus divided wit is
+                            from Egypt by 300 miles of son, is not in the legal or any other sense
+                            of the word, "pays limitropbe" and that being the one no general meeting
+                            of shareholders, can legally improve on other shareholders, or on the
+                            public, the las applying to limited liability</p>
+                        <p>The only legal effect which such a meeting could have, would be to make
+                            the shareholders</p>
+                        <cb n="4"/>
+                        <p>who approved this measure, jointly liable, with the directors for the one
+                            and that liability would not in my opinion be a limit</p>
+                        <p>ODO While thanking Mr. Duke Baker for bis reference to my latter of the
+                            28 inat I beg to say that it does not in any way reply to the question I
+                            raised, and I should be glad if any of your readers would do so, and
+                            such a reply would interest many people in this country besides
+                            myself</p>
+                        <p>OBSERVER Alexandria, January 28</p>
+                    </div>
+                </div>
+                <div type="item">
+                    <head>KHEDIVIAL SPORTING CLUB</head>
+                    <p>SECOND WINIER MEETING</p>
+                    <p>After two of the worst days we have ever experienced in Egypt we rejoiced
+                        greatly in a really fine day for the races. On the 26th the pliance of
+                        racing looked poor, but the come dried in wondertel way and the going w
+                        excellent-josta we bit soft in few places but nothing to matter the times
+                        were naturally not as fast a s a hard course</p>
+                    <p>Sir Eldos aod Lady Gorst were present and the attendance was decidedly
+                        good.</p>
+                    <p>The fields in some of the race dwindleri down rather, but we witnessed some
+                        very good finishes in the first race the starter pony broke loome and nosed
+                        &amp; little diversion Nleyban ran a good noe, this is his second win this
+                        season on this coure He is evidently</p>
+                    <p>good pony - Add we wish his sporting owners further lack with him. The Grand
+                        National Steeplechase was most interesting We are sorry to hear that Capt.
+                        Drake di
```

Comment:
I can find many dates back to 1903 and 1904. Please make sure that you update your meatures.